src/mysql_query.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# igcollect - PostgreSQL query results
+#
+# This script executes a single query and prints the results.  The columns
+# returned by the query are going to be appended to the given prefix.
+# The query must return numeric values.
+#
+# Copyright (c) 2016, InnoGames GmbH
+#
+from __future__ import print_function
+
+import MySQLdb
```

Comment:
Please use `mysql.connector` instead from Oracle.  MySQLdb is not available for Python 3, and I don't think gets much maintenance.